# **The early transcriptional and post-transcriptional responses to Fluconazole in sensitive and resistant *Candida albicans***

**Irene Stevens<sup>1</sup>, Gerald Silao Fitz<sup>2</sup>, Susanne Huch<sup>1</sup>, Kicki Ryman<sup>2</sup>, Honglian Liu<sup>1</sup>, Adriana Carvajal Jimenez<sup>1</sup>, Per O. Ljungdahl<sup>2</sup>, Vicent Pelechano<sup>1</sup>**


<sup>1</sup> Scilifelab, Department of Microbiology, Tumor and Cell Biology, Karolinska Universitet, Solna, Sweden.     
<sup>2</sup> Science for Life Laboratory (SciLifeLab), Department of Molecular Biosciences, Stockholm University, Stockholm, Sweden. 


# **Abstract**
Candida albicans is a leading cause of fungal infections in immunocompromised patients. Management of candidemia relies on a few antifungal agents, with fluconazole being first line therapy. The emergence of fluconazole-resistant strains highlights the pressing need to improve our molecular understanding of the drug response mechanisms. By sequencing the 5’P mRNA degradation intermediates, we show that co-translational mRNA decay is common in C. albicans and characterize how in vivo 5´-3´ exonuclease degradation trails the last translating ribosome. Thus, the study of the 5'P mRNA degradome (5PSeq) offers a simple and affordable way to measure ribosome dynamics and identify codon specific ribosome stalls in response to drugs and amino acid deprivation. Building upon this, we combine RNA-Seq and 5PSeq to study the early response of sensitive and resistant C. albicans isolates to fluconazole. Our results highlight that transcriptional responses, rather than changes in ribosome dynamics, are the main driver of Candida resistance to fluconazole. 

# **Data**
Count tables from the fivepseq package are available in the [Data](https://github.com/irenestevens8/Candida_degradome/tree/main/Data) folder. 

# **Scripts for reproduction of manuscript figures**
All manuscript figures may be reproduced from the R scripts within this repository, with the exception of those not prepared in R.


## **Main Figures**

**Figure 1**: [Rscript Markdown](https://github.com/irenestevens8/Candida_degradome/blob/main/Figures/Figure-1.md).     
**Figure 2**: [Rscript Markdown](https://github.com/irenestevens8/Candida_degradome/blob/main/Figures/Figure-2.md).  
**Figure 3**: [Rscript Markdown](https://github.com/irenestevens8/Candida_degradome/blob/main/Figures/Figure-3.md).                    
**Figure 4**: [Rscript Markdown](https://github.com/irenestevens8/Candida_degradome/blob/main/Figures/Figure-4.md).    

## **Additional analyses pertaining to the study**

[Differential Expression Analysis](https://github.com/irenestevens8/Candida_degradome/blob/main/Figures/Differential_Gene_Expression_analysis.R).          
[Functional Annotation Analysis](https://david.ncifcrf.gov/)

# **Supplementary Tables** 

Table S1: [Candida albicans strains used in study](https://github.com/irenestevens8/Candida_degradome/blob/main/Supplementary%20Files/Table%20S1.xlsx)        

Table S2: [Sequencing and alignment quality metrics](https://github.com/irenestevens8/Candida_degradome/blob/main/Supplementary%20Files/Table%20S2.xlsx) 

Table S3: [Differentially expressed genes ](https://github.com/irenestevens8/Candida_degradome/blob/main/Supplementary%20Files/Table%20S3.xlsx)  

Table S4: [Enriched DAVID Terms, related to Figures 2 and 4](https://github.com/irenestevens8/Candida_degradome/blob/main/Supplementary%20Files/Table%20S4.xlsx)  


# **Contact** 
For questions regarding the 5'P sequencing, please contact:                                        
[Vicente Pelechano](vicente.pelechano.garcia@ki.se)   

For questions regarding the strains used in this study, please contact:                      
[Per O. Ljungdahl](per.ljungdahl@scilifelab.se)                               
                         

# **Links to relevant websites** 
[Fivepseq package for Candida ](https://github.com/irenestevens8/fivepseq/tree/Candida)           
[Fivepseq package User Guide](https://fivepseq.readthedocs.io/en/latest/)                             
[Pelechano Lab website](https://pelechanolab.com/)                     
[Per Ljundahl lab website ](https://www.scilifelab.se/researchers/per-o-ljungdahl/)            
