# **The co-translational degradome provides a readout of Fluconazole response in Candida albicans**

**Irene Stevens<sup>1</sup>, Gerald Silao Fitz<sup>2</sup>, Susanne Huch<sup>1</sup>, Kicki Ryman<sup>2</sup>, Per O. Ljungdahl<sup>2</sup>, Vicente Pelechano<sup>1</sup>**


<sup>1</sup> Science for Life Laboratory, Department of Microbiology, Tumor and Cell Biology, Karolinska Universitet, Solna, Sweden

<sup>2</sup> Science for Life Laboratory, Department of Molecular Biosciences, The Wenner-Gren Institute, Stockholm University, 106 91 Stockholm, Sweden


# **Abstract**
Candida albicans is a leading cause of nosocomial bloodstream infections (BSIs) in high-risk hospital units (e.g. neonatal and intensive care units (ICUs)). Management of candidemia relies on a few antifungal agents, with Fluconazole being safest and first line therapy. Despite its efficacy, the increase in Fluconazole-resistant infections caused by Candida species (spp.) (2) poses a significant challenge to its continual use and highlights the need for improved molecular understanding of drug response. Here we show widespread co-translational decay occurs in Candida albicans and derive Fluconazole-specific degradation signatures. Furthermore, we show that ribosome positional occupancy at nucleotide-, amino acid- codon resolution provides a readout of Fluconazole-induced cellular stress. Finally, we show that frame protection patterns in signature genes are associated with phenotypic response to Fluconazole.

# **Data**
Count tables from the fivepseq package are available in the [Data](https://github.com/irenestevens8/Candida_degradome/tree/main/Data) folder. 

# **Scripts for reproduction of manuscript figures**
All manuscript figures may be reproduced from the R scripts within this repository, with the exception of those not prepared in R.


## **Main Figures**

**Figure 1**: [Rscript markdown](https://github.com/irenestevens8/Candida_degradome)

**Figure 2**: [Rscript markdown](https://github.com/irenestevens8/Candida_degradome)

**Figure 3**: [Rscript markdown](https://github.com/irenestevens8/Candida_degradome)

**Figure 4**: [Rscript markdown](https://github.com/irenestevens8/Candida_degradome)

## **Additional scripts pertaining to the study**

[Differential Expression Analysis](https://github.com/irenestevens8/Candida_degradome)

[Functional Annotation Analysis](https://david.ncifcrf.gov/)

# **Supplementary Tables** 

Table S1: [Read and mapping quality metrics](https://github.com/irenestevens8/Candida_degradome/tree/main/Supplementary%20Tables/Table%20S1.xlsx)

Table S2: Differentially degraded genes and corresponding abundance and frame protection.

# **Contact** 
For all questions regarding this study, please contact:

[Vicente Pelechano](vicente.pelechano.garcia@ki.se) 

# **Links to relevant websites** 
[Fivepseq package for Candida ](https://github.com/irenestevens8/fivepseq/tree/Candida)
[Fivepseq package User Manual](https://fivepseq.readthedocs.io/en/latest/) 
[Pelechano Lab website](https://pelechanolab.com/)
[Per Ljundahl lab website ](https://www.scilifelab.se/researchers/per-o-ljungdahl/)
